dspace-server-webapp/src/main/java/org/dspace/app/rest/submit/step/UploadStep.java

```diff
@@ -134,6 +138,26 @@ public ErrorRest upload(Context context, SubmissionService submissionService, Su
             bf = bitstreamFormatService.guessFormat(context, source);
             source.setFormat(context, bf);
 
+            // Set a default access condition for a newly created bitstream.
+            if (configurationService.getBooleanProperty("uclouvain.bitstream.usedefault.accesstype", true)) {
+                ResourcePolicy rp = authorizeService.createResourcePolicy(
+                    context,
+                    source,
+                    EPersonServiceFactory.getInstance().getGroupService().findByName(context, Group.ANONYMOUS),
+                    null,
+                    Constants.READ,
+                    ResourcePolicy.TYPE_CUSTOM,
+                    UCLouvainAccessStatusHelper.OPEN_ACCESS,
+                    null,
+                    null,
+                    null
+                );
+                // We need to update the resource policies of the bitstream.
+                // This is mandatory so that the returned bitstream has the correct access type.
+                // Maybe there is a better way (Context.reload() and bitstreamService.find() does not work)
+                source.getResourcePolicies().add(rp);
+            }
```

Comment:
* `bitstream.upload.default.accesstype.enabled`
* be better to use a config property for default access type instead of "OpenAccess". Something like a string or better a ref to the class constant (using bean ? much more complicated)